### PR TITLE
Added Proposal for Team3

### PR DESCRIPTION
**Pull request by vrobweis**
Proposal for Team 3's project, Game Getter.

### CODE REVIEW

File: P2-Proposals/Proposal.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# Game-Getter
+
+#### Overview
+We are automating the way we find new videogames. This project will be able to sign into both Steam on Epic Games stores and identify which games are free or on sale. Then, it will able to gather the potential games from both stores to a single location, where the user will then be able to select which games to purchase or download for free. Lastly, the robot will fill in the user's payment information, complete the purchase, and start the download. 
+
+#### What we are automating
+- Opening multiple applications (Steam & Epic Games Launcher)
+- Sign in using user's credentials
+- Find free games and scape data about the game
```
Comment (mariellenolasco) [inline P2-Proposals/Proposal.md]: What is data scaping?

File: P2-Proposals/Proposal.md [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+- Create excel sheet of potential games from both stores
+- Fill in payment informaiton and complete purchase
```
Comment (mariellenolasco) [inline P2-Proposals/Proposal.md]: Fix typos

File: P2-Proposals/Proposal.md [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+
+[Process Flow Diagram](https://app.creately.com/diagram/eb7IAHt1KF4/view)
```
Comment (kaurrevature) [inline P2-Proposals/Proposal.md]: How would you check for all the launchers?
Is there a generic process to install all games without human intervention?

Comment (vrobweis) [reply P2-Proposals/Proposal.md]: The launchers we intend to support all show up in the Windows Programs and Features section of the control panel. 
That means that we can (and actually already have) made code that allows us to check whether the launchers in question are installed, independent of their file path. 

If by generic, you mean a command or some code we can invoke that does it without RPA, no. But for UI navigation, installation of games in the most popular launchers is pretty streamlined, if not simple, with the exception of picking where you want to install your game, which is why that would be left to user choice.

**Pull request closed without merging**